funcx/executors/high_throughput/interchange.py

```diff
@@ -654,17 +681,35 @@ def start(self, poll_period=None):
                 if manager not in self._ready_manager_queue:
                     logger.warning("[MAIN] Received a result from a un-registered manager: {}".format(manager))
                 else:
+                    try:
+                        logger.debug("[MAIN] Trying to unpack ")
+                        manager_report = Message.unpack(b_messages[0])
+                        logger.info(f"[MAIN] Got manager status report: {manager_report.task_statuses}")
+                        self.task_status_deltas.update(manager_report.task_statuses)
+                        b_messages = b_messages[1:]
+                    except Exception:
+                        pass
```

Comment:
it is better to log something here?

Reply:
Well it's not really an error, so I didn't.  This ugliness is because this PR isn't supposed to revamp all of the messaging related things, so it's in a mixed state right now.  So it just tries interpreting it as the new type, and if that fails it uses the old way.  Once the messaging is more uniform, we will just be able to directly tell what kind of message we are receiving and we won't need to do this.

Reply:
Yea, I understood it is not an error. I just think it is good to log that it is not a HB message---for ease of future development.

Reply:
ZZ, this won't be a good idea here because we have 2 types of messages, and we see the exception if it isn't of the Message type. So you'd fill up the logs with garbage errors.